### PR TITLE
Session management

### PR DESCRIPTION
**Pull request by L3nn0x**
this implements #47

---
This change is [Reviewable](https://reviewable.io/reviews/dev-osrose/osirose-new/112)

### CODE REVIEW

File: Database/osirose.sql [hunk 1]
```diff
@@ -33,7 +33,7 @@ CREATE TABLE `accounts` (
   `online` tinyint(1) DEFAULT '0',
   `login_count` int(11) DEFAULT '0',
   `lastip` varchar(15) DEFAULT '0.0.0.0',
-  `lasttime` int(11) DEFAULT '0',
+  `lasttime` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
```
Comment (RavenX8) [inline Database/osirose.sql:36]: This field is named incorrectly. This is the last time the account logged in. When I was talking about using a timestamp for the sessions I meant actually adding the timestamp to the sessions table.

Comment (L3nn0x) [reply Database/osirose.sql:36]: But there's already a timestamp in the session table, so I thought you wanted a timestamp as well for the last time the account logged in

**Comment by RavenX8** [comment]
Reviewed 3 of 9 files at r1, 4 of 4 files at r2, 6 of 6 files at r4.
Review status: 13 of 14 files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[src/char/src/main.cpp, line 125 at r4](https://reviewable.io:443/reviews/dev-osrose/osirose-new/112#-KtD1CwhO3FlTNikJND-:-KtD1CwhO3FlTNikJND0:bfe23lk) ([raw file](https://github.com/dev-osrose/osirose-new/blob/d3ede68718b9fc6808e97461caf87ffe3e6d28d8/src/char/src/main.cpp#L125)):*
> ```cpp
> }*/
> 
> /*void updateSessions() {
> ```

Should remove this code if it's not being used.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/dev-osrose/osirose-new/112)*

**Comment by RavenX8** [comment]
Reviewed 2 of 2 files at r5.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks broke.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/dev-osrose/osirose-new/112)*

**Pull request merged**